### ISSUE TITLE
`BufferCollection.concat()` method

### ISSUE THREAD

**Issue by overlookmotel** [Daninet/buffer-collection#1]
Hi.

This module looks great. I was about to start writing basically exactly the same thing, and then I came across yours - and it's all tested and everything!

I was wondering if there's a reason why `BufferCollection.concat()` isn't supported?

It would be useful to be able to concatenate multiple `BufferCollection`s to create a new `BufferCollection`. e.g.:

```js
const b1 = new BufferCollection().push( [65, 66] ).push( 67 );
const b2 = new BufferCollection().push( [68, 69] ).push( 70 );

assert( b1.count == 2 );
assert( b1.length == 3 );
assert( b1.toString() == 'ABC' );

assert( b2.count == 2 );
assert( b2.length == 3 );
assert( b2.toString() == 'DEF' );

const b3 = BufferCollection.concat( [b1, b2] );

assert( b3.count == 4 );
assert( b2.length == 6 );
assert( b3.toString() == 'ABCDEF' );
```

At first glance, this doesn't look enormously hard to implement, so I wondered if there was a reason why you'd decided not to?

**Comment by Daninet** [comment]
Hey.

Thank you for pointing this out.
It wasn't implemented because I didn't need it in my projects.
But I have implemented it now, in version 1.1.0.

**Comment by overlookmotel** [comment]
Holy cow you are fast! Thank you.
